modules/ingester/instance_search_test.go

```diff
@@ -145,6 +147,94 @@ func TestInstanceSearch(t *testing.T) {
 	checkEqual(t, ids, sr)
 }
 
+func TestInstanceSearchTags(t *testing.T) {
```

Comment:
Was wondering if we could write a test where we return 500 once the tag-values limit is exceeded

Reply:
added.